src/helpers.ts

```diff
@@ -162,3 +164,20 @@ export function toIterable<T>(i: any): any {
     }
     return [i]; // a simple value
 }
+
+/**
+ * Merge multiple iterators or iterables in to a single iterable.
+ * Merged inputs are iterated over in the order in which they were specified.
+ *
+ * Note that if you concatenate asynchronous iterables with a synchronous pipeline, they will be processed as simple values.
+ *
+ * @see [[concat]]
+ * @category Sync+Async
+ */
+export function mergeIterables<Ts extends [...unknown[]]>(
```

Comment:
There's probably a better name for this.

Reply:
If you are looking for a shorter syntax, that would be just one function, without `pipe` at all. Other than that, consistency of using the library is important, I do not want to create deviations from that.

Reply:
@RebeccaStevens  You can add locally, if you use it often enough:

```ts
function merge<A>(i1: A[]): Iterable<A>;
function merge<A, B>(i1: A[], i2: B[]): Iterable<A | B>;
function merge<A, B, C>(i1: A[], i2: B[], i3: C[]): Iterable<A | B | C>;

function merge(...values: unknown[]): unknown {
    return pipe([], concat(...values));
}
```